dist/api_data.py

```diff
@@ -1182,6 +1186,13 @@ def __cmp__(self, other):
         choices=['background', 'off', 'on']),
 ]),
 
+'WT_SESSION.prepare_transaction' : Method([
+    Config('prepare_timestamp', '', r'''
+        set the prepare timestamp for the updates of the current transaction.
+        The supplied value should not be older than any active read timestamps.
+        See @ref transaction_timestamps'''),
```

Comment:
I'd like it if the description explains that the configuration option is required.